README.md

```diff
@@ -40,9 +41,153 @@ Spline aims to fill a big gap within the Apache Hadoop ecosystem. Spark jobs sho
 
 ---
 
-    Copyright 2017 ABSA Group Limited
+## Get Spline
+To get started, you need to get a minimal set of Spline's moving parts - 
+a server, an admin tool and a client Web UI to see the captured lineage.
+
+There are two ways how to do it:
+
+#### Download prebuild Spline artifacts from the Maven repo
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4568241)